argmin-math/src/primitives/minmax.rs

```diff
@@ -49,3 +49,27 @@ make_minmax!(Complex<isize>);
 make_minmax!(Complex<usize>);
 
 // TODO: tests!!!
```

Comment:
Please remove this line.

Reply:
I've updated it,it should be good,let me know if there's anything else to be done